crates/fuel-core/src/service/config.rs

```diff
@@ -63,13 +77,48 @@ impl Config {
             ),
             block_producer: Default::default(),
             block_executor: Default::default(),
+            block_importer: Default::default(),
             #[cfg(feature = "relayer")]
             relayer: Default::default(),
             #[cfg(feature = "p2p")]
             p2p: P2PConfig::<NotInitialized>::default("test_network"),
             consensus_key: Some(Secret::new(default_consensus_dev_key().into())),
         }
     }
+
+    pub fn poa_config(&self) -> anyhow::Result<fuel_core_poa::Config> {
```

Comment:
Would using `impl From<Config> for fuel_core_poa::Config` instead be more idiomatic rust?